### ISSUE TITLE
Mobile Viewing Problem

### ISSUE THREAD

**Issue by russleyshaw** [siggame/webserver#120]
A competitor told me that he was having trouble navigating the site in his iPhone 4. There were issues scrolling and specifically had trouble in inviting his team members.

**Comment by michaelwisely** [comment]
It's probably related to the sidebar. We can take a quick look at it
tonight, otherwise it can wait until we have more time to fiddle with it.

**Comment by jonsimington** [comment]
Yup, the sidebar isn't scrollable on mobile.  The "invite derp to my team" button is just off screen.

**Comment by michaelwisely** [comment]
oooooooh dang. we should look at that today.

**Comment by michaelwisely** [comment]
@jonsimington is this still an issue? I don't remember if/how we resolved it.
